crates/holochain/src/fixt.rs

```diff
@@ -255,7 +262,14 @@ fixturator!(
         let keystore = holochain_state::test_utils::test_keystore();
         tokio_helper::block_forever_on(async {
             fake_genesis(authored_env.env(), dht_env.env(), keystore.clone()).await.unwrap();
-            HostFnWorkspace::new(authored_env.env(), dht_env.env(), cache.env(), keystore, Some(fixt!(AgentPubKey, Predictable, get_fixt_index!()))).await.unwrap()
+            HostFnWorkspace::new(
+                authored_env.env(),
+                dht_env.env(),
+                cache.env(),
+                keystore,
+                Some(fixt!(AgentPubKey, Predictable, get_fixt_index!())),
+                Arc::new(fixt!(DnaDef))
```

Comment:
Won't this lead to dna defs that are randomly in the future to the data that's committed?

Reply:
Good point, I'll set them all to 0.